### ISSUE TITLE
LabelFont command for color not recognised

### ISSUE THREAD

**Issue by Cedrok** [ditrit/OGrEE-CLI#77]
The syntax is `[name]:labelFont=color@[color] ` with [color] an hexadecimal code
For example, this should work:
```
cedrok@OGrEE3D:/Physical$> CED.BETA.A.R1.B09:labelFont=color@123456
OGREE: Unrecognised command!
```

**Comment by hchybz** [comment]
interpreter/lexer.nex

```
/[0-9a-fA-F]/ { printToks("TOK_HEX");
/[0-9a-fA-F]{6}/ { printToks("TOK_HEXCOLOR"};
```

**Comment by denizul** [comment]
Added a fix for this d1da2aed on 'hex' branch
